app/Phragile/ActionHandler/SprintStoreActionHandler.php

```diff
@@ -41,6 +46,34 @@ private function validate()
 		}
 	}
 
+	private function trySprintCreationFromPhabricatorID()
+	{
+		if ($this->previousActionFailed()
+			|| !is_numeric($this->sprint->title)
+			|| !is_int($this->sprint->title + 0)) return;
```

Comment:
You sure the last part of this works correctly in all cases? I've used `ctype_digit` in the past. It's a bit odd that you do not know here if the thing is a string or an int.